src/components/views/elements/DeviceVerifyButtons.js

```diff
@@ -65,7 +65,8 @@ export default React.createClass({
                         <ul>
                             <li><label>Device name:</label> <span>{ this.state.device.getDisplayName() }</span></li>
                             <li><label>Device ID:</label>   <span><code>{ this.state.device.deviceId}</code></span></li>
-                            <li><label>Device key:</label>  <span><code><b>{ this.state.device.getFingerprint() }</b></code></span></li>
+                            <li><label>Device fingerprint:</label>  <span><code><b>{ this.state.device.getFingerprint() }</b></code></span></li>
```

Comment:
I guess I find it confusing that we have different names for it different places.  I'd prefer (in code) if we referred to them as "signing key" and something else (maybe "stream key"?).  "Fingerprint" seems to imply that it's a partial key, or something one could use to quickly verify matches, not that it is our primary source of trust for this user.

Would you be opposed to me making a different PR with "fingerprint" replaced by "signing key" or something similar?